tracer/build/_build/docker/smoke.windows.nuget.dockerfile

```
@@ -14,7 +14,7 @@ COPY ./test/test-applications/regression/AspNetCoreSmokeTest/ .
 ARG PUBLISH_FRAMEWORK
 RUN dotnet restore "AspNetCoreSmokeTest.csproj" \
     && dotnet nuget add source "c:\src\artifacts" \
-    && dotnet add package "Datadog.Monitoring.Distribution" --prerelease \
+    && dotnet add package "Datadog.Trace.Bundle" --prerelease \
```

No need for the prerelease anymore

```suggestion
    && dotnet add package "Datadog.Trace.Bundle" \
```